sparse_strips/vello_hybrid/src/render.rs

```diff
@@ -306,25 +333,44 @@ impl Renderer {
                         },
                         wgpu::BindGroupEntry {
                             binding: 1,
-                            resource: config_buf.as_entire_binding(),
+                            resource: config_buffer.as_entire_binding(),
                         },
                     ],
                 });
                 (alphas_texture, render_bind_group)
             } else {
-                let resources = self.resources.as_ref().unwrap();
+                let resources = self.resources.as_ref().expect("Resources not initialized");
                 (
                     resources.alphas_texture.clone(),
                     resources.render_bind_group.clone(),
                 )
             };
+
             self.resources = Some(GpuResources {
                 strips_buffer,
                 alphas_texture,
                 render_bind_group,
+                config_buffer,
             });
         };
 
+        // Update config buffer if dimensions changed and config buffer exists.
+        // We don't need to initialize a new config buffer because it's fixed size (uniform buffer).
+        if needs_new_config && self.resources.is_some() {
```

Comment:
How would `self.resources` ever be `None` here? We unwrap it in the next statement in this block anyway.
It looks to me that `needs_new_config` will still be true even if the buffer was just created (although I do note that this is only an issue on the first frame, so isn't a huge concern).